src/Sfa.Roatp.Indexer.ApplicationServices/Monitoring/MonitoringService.cs

```diff
@@ -0,0 +1,36 @@
+﻿using System.Net;
+using System.Net.Http;
+using System.Threading.Tasks;
+using Sfa.Roatp.Indexer.ApplicationServices.Settings;
+using SFA.DAS.NLog.Logger;
+
+namespace Sfa.Roatp.Indexer.ApplicationServices.Monitoring
+{
+    public class MonitoringService : IMonitoringService
+    {
+        private readonly IMonitoringSettings _monitoringSettings;
+        private readonly ILog _logger;
+
+        public MonitoringService(IMonitoringSettings monitoringSettings, ILog logger)
+        {
+            _monitoringSettings = monitoringSettings;
+            _logger = logger;
+        }
+
+        public void SendMonitoringNotification()
+        {
+            if (string.IsNullOrEmpty(_monitoringSettings.StatusCakeUrl)) return;
```

Comment:
New requirement, can you split the url on ; and send to multiple endpoints

```c#
foreach(var url in _monitoringSettings.StatusCakeUrl.Split(';'){
    ...
}
```
![image](https://cloud.githubusercontent.com/assets/39366/26404117/a4f146e4-4088-11e7-84c0-e4079dd7c99b.png)